eng/service-templates/ProductConstructionService/provision.bicep

```diff
@@ -741,6 +762,17 @@ resource subscriptionTriggererStorageQueueAccess 'Microsoft.Authorization/roleAs
     }
   }
 
+// allow redis cache read / write access to the service's identity
+resource redisCacheBuiltInAccessPolicyAssignment 'Microsoft.Cache/redis/accessPolicyAssignments@2024-03-01' = {
+    name: guid(subscription().id, resourceGroup().id, 'pcsDataContributor')
+    parent: redisCache
+    properties: {
+        accessPolicyName: 'Data Owner'
+        objectId: pcsIdentity.properties.principalId
```

Comment:
Did you test this out? I'm curious if it's expecting the principalId here

Reply:
Yep, tested the redis deployment + the role assignment, worked fine

Reply:
nice, thanks!